### ISSUE TITLE
Connect to Cassandra without passing user credentials.

### ISSUE THREAD

**Issue by romulogoncalves** [NLeSC/dask-cassandra-loader#9]
When we pass `username=None`, the code crashes in 
```
~/.local/lib/python3.6/site-packages/dask_cassandra_loader/loader.py in load_data(self, cassandra_connection, ca_loading_query)
    515                                                     cassandra_connection.session.cluster.contact_points,
    516                                                     self.keyspace,
--> 517                                                     cassandra_connection.auth.username,
    518                                                     cassandra_connection.auth.password)
```

The code needs to be more robust.

**Comment by romulogoncalves** [comment]
Covered by 872a6a5

**Comment by romulogoncalves** [comment]
Solved by #11
